activefires_pp/tests/test_fires_filtering.py

```diff
@@ -278,10 +281,50 @@ def test_general_national_fires_filtering(get_global_mask, setup_comm, get_confi
 
     store_geojson.assert_called_once()
     get_output_msg.assert_called_once()
-    get_global_mask.call_count == 2
+    assert get_global_mask.call_count == 2
 
     assert isinstance(result, pd.core.frame.DataFrame)
     assert len(result) == 1
     np.testing.assert_almost_equal(result.iloc[0]['latitude'], 59.52483368)
     np.testing.assert_almost_equal(result.iloc[0]['longitude'], 17.1681633)
     assert outmsg == ["my fake output message"]
+
+
+@pytest.mark.usefixtures("fake_national_boarders_shapefile")
+@patch('socket.gethostname')
+@patch('activefires_pp.post_processing.read_config')
```

Comment:
why does `read_config` need to be patched? could we use a minimal configuration files here instead?

Reply:
Yes, right, fixed